<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudienceView</class>
 <widget class="QDialog" name="AudienceView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1384</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>1081</width>
     <height>20</height>
    </rect>
   </property>
   <property name="midLineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>0</y>
     <width>20</width>
     <height>491</height>
    </rect>
   </property>
   <property name="midLineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>480</y>
     <width>1261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="midLineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600; color:#aa0000;&quot;&gt;Win Predictions:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>21</width>
     <height>61</height>
    </rect>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="midLineWidth">
    <number>5</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>511</width>
     <height>20</height>
    </rect>
   </property>
   <property name="midLineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>1080</x>
     <y>0</y>
     <width>311</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Latest Throws :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>100</y>
     <width>20</width>
     <height>391</height>
    </rect>
   </property>
   <property name="midLineWidth">
    <number>2</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="MainStatsLable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>600</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;Stats :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="StatisticsDisplay">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>530</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="Player2Stats">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>710</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="WinPercentages">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>700</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="CurrentPlayer">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>710</y>
     <width>241</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="NumberOf180s">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>530</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="PersonalStats">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>610</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="MatchStats">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>610</y>
     <width>261</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="RankedStats">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>520</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="BothP1AndP2Stats">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>610</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>550</y>
     <width>351</width>
     <height>201</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../../../Pictures/RaidLOGO.PNG</pixmap>
   </property>
  </widget>
  <zorder>BothP1AndP2Stats</zorder>
  <zorder>line</zorder>
  <zorder>label</zorder>
  <zorder>line_4</zorder>
  <zorder>line_5</zorder>
  <zorder>tableWidget</zorder>
  <zorder>label_2</zorder>
  <zorder>line_6</zorder>
  <zorder>MainStatsLable</zorder>
  <zorder>StatisticsDisplay</zorder>
  <zorder>Player2Stats</zorder>
  <zorder>WinPercentages</zorder>
  <zorder>CurrentPlayer</zorder>
  <zorder>NumberOf180s</zorder>
  <zorder>line_2</zorder>
  <zorder>PersonalStats</zorder>
  <zorder>MatchStats</zorder>
  <zorder>RankedStats</zorder>
  <zorder>label_3</zorder>
  <zorder>line_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
